sky/packages/sky/lib/src/fn3/input.dart

```diff
@@ -100,12 +112,17 @@ class InputState extends State<Input> {
       width: focused ? 2.0 : 1.0
     ));
 
-    Container input = new Container(
+    Container container = new Container(
       child: new Stack(textChildren),
       padding: _kTextfieldPadding,
       decoration: new BoxDecoration(border: focusHighlight)
     );
 
+    SizeObserver input = new SizeObserver(
+      child: container,
+      callback: _handleContainerSizeChanged
+    );
+
     return new Listener(
       child: input,
```

Comment:
I'd just inline container and input into the return statement here. Not sure why it was done separately before. It's easier to visualise the tree if it's all inline.